packages/node_modules/@ciscospark/widget-message-meet/src/container.js

```diff
@@ -69,41 +101,68 @@ export class MessageMeetWidget extends Component {
     const {
       avatar,
       spark,
-      sparkState,
-      user
+      user,
+      widgetMessageMeet
     } = props;
-    const {
-      toPersonEmail,
-      toPersonId
-    } = this.props;
 
     const toUser = user.get(`toUser`);
+    const showActivityMenu = widgetMessageMeet.get(`showActivityMenu`);
+    const showMessage = widgetMessageMeet.get(`activityType`) === ACTIVITY_TYPE_MESSAGE;
+    const showMeet = widgetMessageMeet.get(`activityType`) === ACTIVITY_TYPE_MEET;
+    // Don't remove message component from DOM, just hide its display
+    const messageComponentClassNames = classNames(`message-component-wrapper`, styles.messageComponentWrapper, showMessage ? `` : styles.hiddenComponent);
 
-    return (
-      <div className={classNames(`widget-message-meet`, styles.widgetMessageMeet)}>
-        <div className={classNames(`banner`, styles.banner)} />
-        <div className={classNames(`widget-message-meet-inner`, styles.widgetMessageMeetInner)}>
-          <div className={classNames(`title-bar-wrapper`, styles.titleBarWrapper)}>
+    // Don't load widgets until we have spark loaded and info on the toUser
+    if (spark && toUser && toUser.id) {
```

Comment:
Yea, i'm wondering if we should just pass the `toUser` object to the child widgets and that'll give them all the info they need.